model_analyzer/reports/report_manager.py

```diff
@@ -286,7 +286,7 @@ def _build_detailed_report(self, report_model_config):
             "are the requested configurable plots showing the relationship between "
             "various metrics measured by the Model Analyzer. The above table contains "
             "detailed data for each of the measurements taken for this model config in "
-            "decreasing order of throughput.",
+            "decreasing order of concurrency.",
```

Comment:
is this confirmed via the code?

Reply:
It's actually decreasing order of throughput or latency (which is what I meant to write!) based on online vs. offline mode. I've made the fix to check the mode and print out the correct sorting type used.